[feat] Create desktop view for reset password screen

kaleenchen opened the pull request:
## What's new in this PR 🧑‍🌾
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
-Created desktop view for /forgot-password, email-sent, and /reset-password by adding green div on left side using media queries
-adjusted font sizing to be bigger on desktop (lowkey why is the text obnoxiously big)

Linear: https://linear.app/trap-garden/issue/TG-73/create-desktop-view-for-reset-password-screen
Figma: https://www.figma.com/design/M6GyVRxOPtLugZhLmQwuYF/TRAP-GARDEN-🪴?node-id=3243-19195&m=dev


### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: <img src="place image link here!!!" width="240" height="540">
![Screenshot 2025-04-22 at 1 12 39 AM](https://github.com/user-attachments/assets/457f6c69-3160-4082-921a-f3ba685d4911)
![Screenshot 2025-04-22 at 1 13 19 AM](https://github.com/user-attachments/assets/9782ceff-2a30-477a-b04f-717409585ce4)
![Screenshot 2025-04-22 at 1 13 47 AM](https://github.com/user-attachments/assets/1a4e3593-d2f0-4908-a39f-7e604a0adf9b)




## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

1. expect to see green div and bigger fonts when screen is enlarges to desktop view and is responsive to smaller view
2. @kylezryr idk why the green heading h2 text looks so massive on my screen and sometimes doesnt fit on one line depending on the screen size but I jsut copied the figma (64px)


## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."



## Relevant links
### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'



### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"



CC: @kylezryr

kaleenchen committed:
sprint finished

kylezryr committed:
use airbnb font and adjust spacing

kylezryr commented on app/(auth)/reset-password/page.tsx:
```diff
@@ -89,68 +95,73 @@ export default function ResetPassword() {
+      <SolidGreenDiv />
+      <AuthContentContainer>
+        <StyledForm>
+          <GreenH2>Set new password</GreenH2>
```
I think we're missing the "Enter your password" text below here as shown in the Figma. we may also need to adjust the spacing between the title and the password input

![image](https://github.com/user-attachments/assets/fc79a0d1-b884-4eb9-a479-4522535c953e)

kylezryr reviewed (changes requested):
just 1 change to make! this was really well done, great job! i rebased and made some minor changes because I just imported the AirBnb font which we use for desktop view, so hopefully that makes the text look not as big. i also merged the login/sign up desktop view which affected some of this screen too so I had to make some changes to add padding and spacing here and there. make sure to pull from main and rebase before making changes!

kylezryr committed:
undo rebase error

kaleenchen committed:
sprint finished

# Conflicts:
#	app/(auth)/styles.ts
#	components/TextInput/styles.ts

# Conflicts:
#	app/(auth)/styles.ts

kaleenchen committed:
nit changes to text size, overflow, & scroll

kylezryr committed:
fix zoom in issue
